test/metabase/sample_data_test.clj

```diff
@@ -36,7 +36,6 @@
   {:style/indent 1}
   [[db-binding] & body]
   `(t2.with-temp/with-temp [Database db# (sample-database-db false)]
-     (sync/sync-database! db#)
```

Comment:
Unrelated, but syncing the database was actually only necessary in one test in this namespace but we were doing it in 5 tests, so this tiny change shaves off ~8 seconds from the test suite locally in a bunch of non-parallel tests, which is amazing. On CI it probably shaves off an hour